Update range-test.mdx on message visability and that gps is optional. (#1904)
meshtastic/Meshtastic: docs/configuration/module/range-test.mdx

import Tabs from "@theme/Tabs";
import TabItem from "@theme/TabItem";

> This module allows you to test the range of your Meshtastic nodes. It requires two nodes, only nodes that have the module enabled will see the range test messages:

Reviewer: ```suggestion
This module allows you to test the range of your Meshtastic nodes and requires two nodes. Only nodes that have the module enabled will display range test messages on their connected client:
```
